locksmithctl/locksmithctl.go

```diff
@@ -216,25 +218,33 @@ func getClient() (*lock.EtcdLockClient, error) {
 		transport.TLSClientConfig = tlsconf
 	}
 
-	cfg := client.Config{
-		Endpoints: globalFlags.Endpoints,
-		Transport: transport,
-		Username:  globalFlags.EtcdUsername,
-		Password:  globalFlags.EtcdPassword,
-	}
+	for _, ep := range globalFlags.Endpoints {
```

Comment:
Can we have a comment somewhere about v2 client, kola tests and that this won't be needed with client v3?